core/shared/src/main/scala/fs2/concurrent/pubsub/TopicStrategy.scala

```diff
@@ -0,0 +1,58 @@
+package fs2.concurrent.pubsub
+
+import fs2.internal.Token
+
+import scala.collection.immutable.{Queue => ScalaQueue}
+
+object TopicStrategy {
```

Comment:
no preference. It seemed to me to noisy to have the topic PubSub implementation and strategy in one place. I am ok to do as you suggest.

Reply:
> From code organization perspective, why define these strategy objects instead of putting them in, e.g., Topic companion?

Yeah I had a similar reaction